src/page/index.ts

```diff
@@ -0,0 +1,10 @@
+export { MenuContentPage } from './menu-content.page';
+export { ProductList } from './product-list.page';
```

Comment:
please add the Page suffix to all page objects in here. That implies renaming each class in its corresponding file.

Reply:
Does all the Classes we make will need to specify the name of the .ts file?